src/coreclr/interpreter/compiler.cpp

```diff
@@ -1162,7 +1181,7 @@ class InterpGcSlotAllocator
         if (allocateNewSlot)
         {
             // Important to pass GC_FRAMEREG_REL, the default is broken due to GET_CALLER_SP being unimplemented
```

Comment:
I think this comment is possibly invalid now that you are making GC_SP_REL work, or it should be updated to explain the new rules